src/main/java/rps/CommandLinePrompt.java

```diff
@@ -47,8 +48,23 @@ public ReplayOption readValidReplayOption() {
         return ReplayOption.of(readInput());
     }
 
+    @Override
+    public void displayChosenMove(Gesture gesture, String name) {
+        write(name + " chose " + gesture.getId() + " - " + gesture.name());
+    }
+
+    @Override
+    public void displayWinner(String winnerName) {
+        write(winnerName + " won");
+    }
+
+    @Override
+    public void displayDraw() {
+        write("Draw");
+    }
+
     private String formatGesturesForPrompt() {
-        Gesture[] gestures = Gesture.values();
+        Gesture[] gestures = values();
```

Comment:
values() is an out of the box method provided from enum.

I could wrap it in a method called 'gestures()' and call 'values()' within it, thus hiding it?